Adding behave example project files (#31)
dequelabs/axe-devtools-html-api-examples: Python/behave/README.md

```sh
cd Python/behave
```

## Binary Reporter

Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) for your operating system and place it in the **_resources_** directory.

## Install Dependencies

Install the dependencies including **axe DevTools API**, **DevTools Behave**, and **DevTools Selenium** for the project.

> **_NOTE:_**
>You need a valid license to use our APIs. For more information, see [Install from Deque’s Agora](https://docs.deque.com/devtools-html/4.0.0/en/py-getting-started) page. After configuring the access to Deque's private registry, you can install the dependencies for this project.

The following command installs all the required dependencies to run this example project.

```sh
pip install -r requirements.txt
```

## Run Tests

The **_features_** directory contains the steps and environment files **`steps.py`** and **`environment.py`** that analyze the page `https://broken-workshop.dequelabs.com/` for accessibility issues.

The following command runs the files in the **_features_** directory.

```sh
behave
```

## Test Results

> The tests generate results in the **_a11y-results_** directory if you have placed the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) for your operating system. Otherwise, they generate only JSON results.

Reviewer: ```suggestion
The tests generate results in the **_a11y-results_** directory if you have installed the [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter). Otherwise, they generate only JSON results.
```
